src/monodroid/jni/android-system.h

```diff
@@ -59,7 +59,7 @@ namespace xamarin { namespace android { namespace internal
 #elif LINUX
 		static constexpr char SYSTEM_LIB_PATH[] = "/usr/lib";
 #elif APPLE_OS_X
-		static constexpr char SYSTEM_LIB_PATH[] = "/Library/Frameworks/Xamarin.Android.framework/Libraries/";
+		static constexpr char SYSTEM_LIB_PATH[] = "/Library/Frameworks/Xamarin.Android.framework/Versions/Current/lib";
```

Comment:
Now this is definitely something I missed when working on the .pkg installer changes. I think we'll need to use the following path (as androidtools should be doing):
```
/Library/Frameworks/Xamarin.Android.framework/Versions/Current/lib/xamarin.android/xbuild/Xamarin/Android/lib/host
```
or 
```
/Library/Frameworks/Xamarin.Android.framework/Versions/Current/lib/xamarin.android/xbuild/Xamarin/Android/lib/host-Darwin
```

That being said I'm wondering how we didn't see more fallout from this earlier, or why the designer seemingly works at all on macOS with packages from master?

Reply:
I don't think the designer integration tests are passing on master: http://build.devdiv.io/2511738

I'm not sure the difference between:
```
/Library/Frameworks/Xamarin.Android.framework/Versions/Current/lib
or
/Library/Frameworks/Xamarin.Android.framework/Versions/Current/lib/xamarin.android/xbuild/Xamarin/Android/lib/host-Darwin
```
`host` is a symlink to `host-Darwin`, so it's OK that androidtools is returning that path, I think.

I see `libmonosgen-2.0.dylib` that is the same file size in both of these locations (none are symlinks). Which one is the _right_ one?

Reply:
`/Library/Frameworks/Xamarin.Android.framework/Versions/Current/lib` should no longer contain the duplicate macOS libraries, what version are you seeing that in?

Reply:
I had a 16.0 pkg installed. I think `host-Darwin` is what should be used here. I updated the commit message for this change, too.